src/views/DigitalCredentials.vue

```diff
@@ -1,213 +1,19 @@
-    return this.digitalCredentialSteps.map(step => { return step.to })
-      .indexOf(`/${Routes.DIGITAL_CREDENTIALS}/${this.$route.name}`)
-  }
-
-  get isDigitalCredentialHome (): boolean {
-    return (this.$route.name === Routes.DIGITAL_CREDENTIALS)
-  }
-
-  get nextStepName (): string {
-    return this.digitalCredentialSteps[this.currentStepIndex + 1]?.text || 'Done'
-  }
-
-  addCredentials (): void {
-    this.$router.push({ path: `${Routes.DIGITAL_CREDENTIALS}/${Routes.DOWNLOAD_WALLET}` })
-  }
-
-  back (): void {
-    this.$router.push({ path: this.digitalCredentialSteps[this.currentStepIndex - 1].to })
-  }
-
-  next (): void {
-    this.$router.push({
-      path: this.digitalCredentialSteps[this.currentStepIndex + 1]?.to ||
-      `/${Routes.DIGITAL_CREDENTIALS}`
-    })
-  }
-
-  cancel (): void {
-    this.$router.push({ path: `/${Routes.DIGITAL_CREDENTIALS}` })
-  }
-
-  async getCredentials (): Promise<void> {
-    const { data } = await LegalServices.fetchCredentials(this.getIdentifier)
-    if (data?.issuedCredentials) {
-      this.issuedCredentials = data.issuedCredentials
-    }
-  }
-
-  async addCredentialInvitation (): Promise<void> {
-    const { data } = await LegalServices.createCredentialInvitation(this.getIdentifier)
-    if (data?.invitationUrl) {
-      this.credentialInvitationUrl = data.invitationUrl
-    }
-  }
-
-  async getCredentialsConnection (): Promise<void> {
-    const connection = await LegalServices.fetchCredentialConnection(this.getIdentifier)
-    if (connection) {
-      this.hasRegisteredWallet = true
-    }
-  }
-
-  async issueCredential (credentialType: DigitalCredentialTypes): Promise<void> {
-    const credentialIssued = await LegalServices.issueCredentialOffer(this.getIdentifier, credentialType)
-    if (credentialIssued) {
-      await this.getCredentials()
-    }
-  }
-
-  // Keep credential data in sync when navigating between routes.
-  // This is required due to the nature of external interactions between mobile device and api
-  @Watch('$route', { immediate: true })
-  async syncCredentials (): Promise<void> {
-    await this.getCredentials()
-
-    // Check for connections on step 2 when there is no registered wallet
-    if (this.issuedCredentials.length > 0) { this.hasRegisteredWallet = true }
-    if (!this.hasRegisteredWallet && this.currentStepIndex === 1) { await this.getCredentialsConnection() }
-
-    // Generate an Invitation URL once only if the user has not registered
-    if (!this.hasRegisteredWallet && !this.credentialInvitationUrl) {
-      await this.addCredentialInvitation()
+import { Component, Mixins } from 'vue-property-decorator'
+import { AllowableActions, Routes } from '@/enums'
+import { AllowableActionsMixin } from '@/mixins'
+
+@Component
+export default class DigitalCredentials extends Mixins(AllowableActionsMixin) {
+  mounted (): void {
+    if (!this.isAllowed(AllowableActions.DIGITAL_CREDENTIALS)) {
+      this.$router.push({ name: Routes.DASHBOARD })
```

Comment:
I'm not sure if this is working -- I can't get to (or stay on) this page on the preview site.

https://business-filings-dev--pr-552-zpnetclq.web.app/BC0871407/

Reply:
Still not working using this URL:
https://business-filings-dev--pr-552-zpnetclq.web.app/FM1038931?accountid=2288

Reply:
I've updated the Mixin logic. It was matching on a route which will only work for the Entity Menu but not when routing to the Digital Credentials feature. This should now work.

Reply:
Yep, fixed now.